routes/remove_route.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// remove a route from a list of routes.
+
+module.exports = function (routes, path) {
+  for (var i = 0; i < routes.length; ++i) {
+    var route = routes[i]
+    if (route.path === path) {
+      routes.splice(i, 1)
+      break
+    }
+  }
```

Comment:
In the fxa-auth-db-server, there is an error [notFound](https://github.com/mozilla/fxa-auth-db-server/blob/fa80cf0c996c84271da88cf73578dd95def96f6e/error.js#L31) that would suit the purposes well. `notFound`'s errno collides with this repo's `gone` errno. Should I bring the error over? Does an error that already exists suit the need?